codemods/use-i18n-hook/example-after.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { translate, Text, useI18n } from 'cozy-ui/transpiled/react'
+
+const ComponentUsingT = () => {
+  const { t } = useI18n()
+
+  return (
+    <Padded>
+      <Title>{t('Title1')}</Title>
+      <Text>{t('Text1')}</Text>
+    </Padded>
+  );
+}
+
+const ComponentUsingTAndF = () => {
```

Comment:
Shouldn't this be EnhancedComponentUsingTAndF or DumbComponentUsingTAndF ?

Reply:
Fixed when re-ran the codemod.